sample-apps/react-native/dogfood/src/components/CallControlsComponent.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+  ChatButton,
+  ChatButtonProps,
+  HangUpCallButton,
+  ReactionButton,
+  ToggleAudioPublishingButton,
+  ToggleCameraFaceButton,
+  ToggleVideoPublishingButton,
+} from '@stream-io/video-react-native-sdk';
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { appTheme } from '../theme';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import { Z_INDEX } from '../constants';
+
+export type CallControlsComponentProps = {
+  chatButton?: ChatButtonProps;
```

Comment:
this prop needs to go or be replaced

Reply:
This was a DF app thing. But anyways I have break the props in two parts and used it now. ✅